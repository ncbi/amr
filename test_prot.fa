>blaTEM-156
MSIQHFRVALIPFFAAFCLPVFAHPETLVKVKDAEDQLGARVGYIELDLNSGKILESFRPEERFPMMSTFKVLLCGAVLS
RVDAGQEQLGRRIHYSQNDLVEYSPVTEKHLTDGMTVRELCSAAITMSDNTAANLLLTTIGGPKELTAFLHNIGDHVTRL
DRWEPELNEAIPNDERDTTMPAAMATTLRKLLTGELLTLASRQQLIDWMEADKVAGPLLRSALPAGWFIADKSGAGERGS
RGIIAALGPDGKPSRIVVIYTTGSQATMDERNRQIAEIGASLIKHW
>blaPDC-114_blast BLAST (100% length, but 1 mismatch)
MRDTRFPCLCGIAASTLLFATTPAIAGEAPADRLKALVDAAVQPVMKANDIPGLAVAISLKGEPHYFSYGLASKEDGRRV
TPETLFEIGSVSKTFTATLAGYALTQDKMRLDDRASQHWPALVGSRFDGISLLDLATYTAGGLPLQFPDSVQKDQAQIRD
YYRQWQPTYAPGSQRLYSNPSIGLFGYLAARSLGQPFERLMEQQVFPALGLEQTHLDVPEAALAQYAQGYGKDDRPLRAG
PGPLDAEGYGVKTSAADLLRFVDANLHPERLDRPWAQALDATHRGYYKVGDMTQGLGWEAYDWPISLKRLQAGNSTPMAL
QPHRIARLPAPQALEGQRLLNKTGSTNGFGAYVAFVPGRDLGLVILANRNYPNAERVKIAYAILSGLEQQGKVPLKR
>blaOXA-436_partial (Should be partial OXA-48 family
MRALALSAVLMVTTMIGMPAVAKEWQENKSWNAHFSEHKTQGVVVLWNENTQQGFTNDLKRANQAFLPASTFKIPNSLIA
LDLGVVKDEHQVFKWDGQTRDIAAWNRDHDLITAMKYSVVPVYQEFARQIGEARMSKMLHAFDYGNEDISGNLDSFWLDG
GIRISATQQIAFLRKLYHNKLHVSERSQRIVKQAMLTEANADYIIRAKTGYSVRIEPKIGWWVGWIELDDNVW
>vanG
MQNKKIAVIFGGNSTEYEVSLQSASAVFENINTNKFDIIPIGITRSGEWYHYTGEKEKILNNTWFEDSKN
LCPVVVSQNRSVKGFLEIASDKYRIIKVDLVFPVLHGKNGENGTLQGIFELAGIPVVGCDTLSSALCMDK
DRAHKLVSLAGISVPKSVTFKRFNEEAAMKEIEANLTYPLFIKPVRAGSSFGITKVIEKQELDAAIELAF
EHDTEVIVEETINGFEVGCAVLGIDELIVGRVDEIELSSGFFDYTEKYTLKSSKIYMPARIDAEAEKRIQ
EAAVTIYKALGCSGFSRVDMFYTPSGEIVFNEVNTIPGFTSHSRYPNMMKGIGLSFSQMLDKLIGLYVE
>gyrA  T86I  Campylobacter
MENIFSKDSDIELVDIENSIKGSYLDYSMSVIIGRALPDARDGLKPVHRRILYAMQNDEAKSRTDFVKSARIVGAVIGRYHPHGDAAVYDALVRMAQDFSMRYPSITGQGNFGSIDGDSAAAMRYTEAKMSKLSHELLKD
IDKDTVDFVPNYDGSESEPDVLPSRVPNLLLNGSSGIAVGMATNIPPHSLNELIDGLLYLLDSKDASLEE
IMQFIKGPDFPTGGIIYGKKGIIEAYRTGRGRVKVRAKTHIEKKTNKDVIVIDELPYQTNKARLIEQIAE
LVKEKQIEGISEVRDESNKEGIRVVIELKREAMSEIVLNNLFKSTTMESTFGVIMLAIHNKEPKIFSLLE
LLNLFLTHRKTVIIRRTIFELQKARARAHILEGLKIALDNIDEVIALIKNSSDNNTARDSLVAKFGLSEL
QANAILDMKLGRLTGLEREKIENELAELMKEIARLEEILKSETLLENLIRDELKEIRSKFDVPRITQIED
DYDDIDIEDLIPNENMVVTITHRGYIKRVPSKQYEKQKRGGKGKLAVTTYDDDFIESFFTANTHDTLMFV
TDRGQLYWLKVYKIPEGSRTAKGKAVVNLINLQAEEKIMAIIPTTDFDESKSLCFFTKNGIVKRTNLSEY
QNIRSVGVRAINLDENDELVTAIIVQRDEDEIFATGGEENLENQEIENLDDENLENEESVSTQGKMLFAV
TKKGMCIKFPLAKVREIGRVSRGVTAIKFKEKNDELVGAVVIENDEQEILSISAKGIGKRTNAGEYRLQS
RGGKGVICMKLTEKTKDLISVVIVDETMDLMALTSSGKMIRVDMQSIRKAGRNTSGVIVVNVENDEVVSI
AKCPKEENDEDELSDENFGLDL
>50S_L22 Campylobacter 50S_L22:A103V
MSKALIKFIRLSSTKARLIAREVQGMNAELAMASLKFMPNKGAKYIANAISSAVANGGFEANEVIVKSCRVDAAAVLKRF
RPRARGSASRIRKPTSHILVEVVKAEVKAEEKKTVAKKTTTTKAPAKKTTSTKKATAKKES

>nimIJ_hmm  WP_027455679.1 NimIJ family nitroimidazole resistance protein [Prevotella brevis]
MSEFREMRRKRQQLTDADSIAVLQKATSGTLALLGDNDYPYAVPISYVYDNGKLYFHSAMAGHKVDAIRR
CNKASFCVIEKDDVRPEKYTTYFRSVIAFGRIEIVEDEAEKRTIMHMMGNRFNPNHDDALQKELESGLAH
MLAIRMDIEHLTGKEAIELVRQRGGN

>aph3pp-Ib_partial_5p_neg  NZ_QKNQ01000001.1 Providencia rettgeri strain Pret_2032, whole genome shotgun sequence  2160922-2162737  150-1527  704-137
IRKLKEPPLNRTNIFFGESHSDWLPVRGGESGDFVFRRGDGHAFAKIAPASRRGELAGERDRLIWLKGRGVACPEVINWQEEQEGACLVITAIPGVPAADLSGADLLKAWPSMGQQLGAVHSLSVDQCPFERRLSRMFGRAVDVVSRNAVNPDFLPDEDKSTPQLDLLARVERELPVRLDQERTDMVVCHGDPCMPNFMVDPKTLQCTGLIDLGRLGTADRYADL

>sul2_partial_3p_neg   NZ_QKNQ01000001.1 Providencia rettgeri strain Pret_2032, whole genome shotgun sequence  2160922-2162737  150-1377  2-667
SSNPDAAPVSSDTEIERIAPVLDALKADGIPVSLDSYQPATQAYALSRGVAYLNDIRGFPDAAFYPQLAKSSAKLVVMHSVQDGQADRREAPAGDIMDHIAAFFDARIAALTGAGIKRNRLVLDPGMGFFLGAAPETSLSVLARFDELRLRFDLPVLLSVSRKSFLRALTGRGPGDVGAATLAAELAAAAGGADFIRTHEPRPLRDGLAVLAALKETARIR

>blaTEM-internal_stop
HFRVALIPFFAAFCLPVFAHPETLVKVKDAEDQLGARVNYIELDLNSGKILESFRPEERFPMMSTFKVLLCGAVLSCVDAG
QEQLGRSIHYSQNDLVEYSPVTEKHLTDGMTLRELCSAAITMSDNTAANLLLTTIGGPKELTA

>qacR-curated_blast  (ADK23698.1)
MKLKDKILGVAKELFIKNGYNATTTGEIVKLSESSKGNLYYHFKTKENLFLEILNIEESKWQEQWKKEQI
KCKTNREKFYLYNELSLTTEYYYPLQNAIIEFYTEYYKTNSINEKMNKLENKYIDAYHVIFKEGNLNGEW
CINDVNAVSKIAANAVNGIVTFTHEQNINERIKLMNKFSQIFLNGLSK

>arsR-suppressed-in-escherichia BAE77793.1  arsR_K-12 As(III)-sensing_metalloregulatory_transcriptional_repressor_ArsR
MSFLLPIQLFKILADETRLGIVLLLSELGELCVCDLCTALDQSQPKISRHLALLRESGLLLDRKQGKWVHYRLSPHIPAW
AAKIIDEAWRCEQEKVQAIVRNLARQNCSGDSKNICS

>emrD3-suppressed-in-vibrio ABQ18953.1
MKTKPSLWLMVIMLMFPQIVETIYSPVLGSIARSFSVSDAQAAQTLSVYFLAFALGVVIWGVLADKWGRRPTMLVGLLIY
GSATFIAMQTDSFTILMLARVFSAFGIAVGSVVTQTILRDVFSGHELRKVFSLMGIGISISPVLGMLLGGQLAFAGGHQL
VFLALFFIALVLFVYNLCQLPETQQVKPKIALGCLVARMFKDRQVLLSALLVALYNVALFSYYQLGAFIFSDLGLDAEQF
GYSGIALGLGSLIGSFLNKTLLAKQVPQRALLLLAALLLIMGTIGVSLTLDSIGFVAAMILVVIAYGMAIPNILSTALVE
YKSQAGSAGALFGLLYYLLIGSGLALTGLVQRLGVVLLMCAGITLLATLARSSHIARLP

>pmrB_C84R
MHFLRRPISLRQRLILTIGAILLVFELISVFWLWHESTEQIQLFEQALRDNRNNDRHIMREIREAVASLIVPGVFMVSLTLFIRYQAVRRITRPLAELQKELEARTADNLTPIAIHSATLEIEAVVSALNDLVSRLTSTLDNERLFTADVAHELRTPLAGVRLHLELLAKTHHIDVAPLVARLDQMMESVSQLLQLARAGQSFSSGNYQHVKLLEDVILPSYDELSTMLDQRQQTLLLPESAADITVQGDATLLRMLLRNLVENAHRYSPQGSNIMIKLQEDDGAVMAVEDEGPGIDESKCGELSKAFVRMDSRYGGIGLGLSIVSRITQLHHGQFFLQNRQETSGTRAWVRLKKDQYVANQI

>nfsA_R15C_K141STOP 
MTPTIELICGHRSICHFTDEPISEAQREAIINSARATSSSSFLQYSSIIRITDKALREELVTLTGGQKHVAQAAEFWVFCADFNRHLQICPDAQLGLAEQLLLGVVDTAMMAQNALIAAESLGLGGVYIGGLRNNIEAVT

