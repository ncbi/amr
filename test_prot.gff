##gff-version 3
##sequence-region contig1 1-50000
contig01	.	gene	101	961	.	+	.	ID=gene1;Name=blaTEM-156
contigX	.	gene	1	501	.	+	.	ID=gene2;Name=nimIJ_hmm
contig02	.	gene	1	1191	.	+	.	ID=gene3;Name=blaPDC-114_blast
contig03	.	gene	101	802	.	+	.	ID=gene4;Name=blaOXA-436_partial
contig04	.	gene	101	1147	.	+	.	ID=gene5;Name=vanG
contig06	.	gene	31	2616	.	+	.	ID=gene6;Name=gyrA
contig07	.	gene	101	526	.	+	.	ID=gene7;Name=50S_L22
contig09	.	gene	1	675	.	-	.	Name=aph3pp-Ib_partial_5p_neg
contig09	.	gene	715	1377	.	-	.	Name=sul2_partial_3p_neg
contig11	.	gene	113	547	.	+	.	Name=blaTEM-internal_stop
contig12	.	gene	71	637	.	+	.	Name=qacR-curated_blast
contig13	.	gene	1	1137	.	+	.	Name=emrD3-suppressed-in-vibrio
contig13	.	gene	1141	1491	.	+	.	Name=arsR-suppressed-in-escherichia
contig14	.	gene	1093	2181	.	+	.	Name=pmrB_C84R
contig16	.	gene	1	423	.	+	.	Name=nfsA_R15C_K141STOP
contig18	.	gene	279	1238	.	+	.	Name=stxA2a_prot
contig18	Protein Homology	CDS	279	1238	.	+	0	Name=stxA2a_prot
contig18	.	gene	1250	1519	.	+	.	Name=stxB2a_prot
contig18	Protein Homology	CDS	1250	1519	.	+	0	Name=stxB2a_prot
