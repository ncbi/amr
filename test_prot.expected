Protein identifier	Contig id	Start	Stop	Strand	Gene symbol	Sequence name	Scope	Element type	Element subtype	Class	Subclass	Method	Target length	Reference sequence length	% Coverage of reference sequence	% Identity to reference sequence	Alignment length	Accession of closest sequence	Name of closest sequence	HMM id	HMM description
blaTEM-156	contig01	101	961	+	blaTEM-156	class A beta-lactamase TEM-156	core	AMR	AMR	BETA-LACTAM	BETA-LACTAM	ALLELEP	286	286	100.00	100.00	286	WP_061158039.1	class A beta-lactamase TEM-156	NF000531.2	TEM family class A beta-lactamase
blaPDC-114_blast	contig02	1	1191	+	blaPDC	PDC family class C beta-lactamase	core	AMR	AMR	BETA-LACTAM	CEPHALOSPORIN	BLASTP	397	397	100.00	99.75	397	WP_061189306.1	class C beta-lactamase PDC-114	NF000422.6	PDC family class C beta-lactamase
blaOXA-436_partial	contig03	101	802	+	blaOXA	OXA-48 family class D beta-lactamase	core	AMR	AMR	BETA-LACTAM	BETA-LACTAM	PARTIALP	233	265	87.92	100.00	233	WP_058842180.1	OXA-48 family carbapenem-hydrolyzing class D beta-lactamase OXA-436	NF012161.0	class D beta-lactamase
vanG	contig04	101	1147	+	vanG	D-alanine--D-serine ligase VanG	core	AMR	AMR	GLYCOPEPTIDE	VANCOMYCIN	EXACTP	349	349	100.00	100.00	349	WP_063856695.1	D-alanine--D-serine ligase VanG	NF000091.3	D-alanine--D-serine ligase VanG
aph3pp-Ib_partial_5p_neg	contig09	1	675	-	aph(3'')-Ib	aminoglycoside O-phosphotransferase APH(3'')-Ib	core	AMR	AMR	AMINOGLYCOSIDE	STREPTOMYCIN	PARTIAL_CONTIG_ENDP	225	275	81.82	100.00	225	WP_109545041.1	aminoglycoside O-phosphotransferase APH(3'')-Ib	NF032896.1	APH(3'') family aminoglycoside O-phosphotransferase
sul2_partial_3p_neg	contig09	715	1377	-	sul2	sulfonamide-resistant dihydropteroate synthase Sul2	core	AMR	AMR	SULFONAMIDE	SULFONAMIDE	PARTIALP	221	271	81.55	100.00	221	WP_001043265.1	sulfonamide-resistant dihydropteroate synthase Sul2	NA	NA
blaTEM-internal_stop	contig11	113	547	+	blaTEM	TEM family class A beta-lactamase	core	AMR	AMR	BETA-LACTAM	BETA-LACTAM	PARTIALP	144	286	50.35	97.22	144	WP_000027057.1	class A broad-spectrum beta-lactamase TEM-1	NA	NA
qacR-curated_blast	contig12	71	637	+	qacR	multidrug-binding transcriptional regulator QacR	plus	STRESS	BIOCIDE	QUATERNARY AMMONIUM	QUATERNARY AMMONIUM	BLASTP	188	188	100.00	99.47	188	ADK23698.1	multidrug-binding transcriptional regulator QacR	NA	NA
emrD3-suppressed-in-vibrio	contig13	1	1137	+	emrD3	multidrug efflux MFS transporter EmrD-3	plus	AMR	AMR	EFFLUX	EFFLUX	EXACTP	379	379	100.00	100.00	379	ABQ18953.1	multidrug efflux MFS transporter EmrD-3	NA	NA
pmrB_C84R	contig14	1093	2181	+	pmrB_C84R	Escherichia colistin resistant PmrB	core	AMR	POINT	COLISTIN	COLISTIN	POINTP	363	363	100.00	99.72	363	WP_001300761.1	two-component system sensor histidine kinase PmrB	NA	NA
nfsA_R15C_K141STOP	contig16	1	423	+	nfsA_R15C	Escherichia nitrofurantoin resistant NfsA	core	AMR	POINT	NITROFURAN	NITROFURANTOIN	POINTP	140	240	58.33	99.29	140	WP_089631889.1	nitroreductase NfsA	NA	NA
nimIJ_hmm	contigX	1	501	+	nimIJ	NimIJ family nitroimidazole resistance protein	core	AMR	AMR	NITROIMIDAZOLE	NITROIMIDAZOLE	HMM	166	165	98.18	76.54	162	WP_005812825.1	NimIJ family nitroimidazole resistance protein	NF000262.1	NimIJ family nitroimidazole resistance protein
